packages/cactus-plugin-ledger-connector-corda/src/main/typescript/plugin-ledger-connector-corda.ts

```diff
@@ -100,6 +138,9 @@ export class PluginLedgerConnectorCorda
       this.prometheusExporter,
       `${fnTag} options.prometheusExporter`,
     );
+
+    this.baseUrl = "https://127.0.0.1:8888/api/v1";
```

Comment:
@adrianbatuto Please do not hardcode the base URL to localhost. Make it a constructor parameter.